packages/scene-composer/src/components/toolbars/common/ItemContainer.tsx

```diff
@@ -59,7 +62,9 @@ export const ItemContainer = React.forwardRef<HTMLDivElement, ToolbarItemContain
     const pointerUp = useCallback(
       (e) => {
         onPointerUp?.(e);
-        onItemClick?.(item);
+        if (!item.isDisabled) {
+          onItemClick?.(item);
+        }
```

Comment:
Nit: Consider adding onItemClick existence to if statement to improve readability.

Reply:
done